core/config/locales/en.yml

```diff
@@ -711,6 +711,7 @@ en:
     add_action_of_type: Add action of type
     add_country: Add Country
     add_coupon_code: Add Coupon Code
+    add_line_item: Add Item
```

Comment:
Is there any process for reporting this new translation to `solidus_i18n`? Perhaps an issue could be created on the `solidus_i18n` repo when translation-changes/new-translation-is-added?